apollo-runtime/src/main/java/com/apollographql/android/impl/RealApolloWatcher.java

```diff
@@ -0,0 +1,83 @@
+package com.apollographql.android.impl;
+
+import com.apollographql.android.ApolloCall;
+import com.apollographql.android.ApolloWatcher;
+import com.apollographql.android.api.graphql.Operation;
+import com.apollographql.android.api.graphql.Response;
+import com.apollographql.android.cache.normalized.Cache;
+import com.apollographql.android.cache.normalized.CacheControl;
+import com.apollographql.android.impl.util.Utils;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+final class RealApolloWatcher<T extends Operation.Data> implements ApolloWatcher<T> {
```

Comment:
Do we need to have `cancel` or `dispose` method, to cancel watching via `RealApolloWatcher` as well not only via `WatcherSubscription `